example-crates/example_cli/Cargo.toml

```diff
@@ -7,11 +7,11 @@ edition = "2021"
 
 [dependencies]
 bdk_chain = { path = "../../crates/chain", features = ["serde", "miniscript"]}
+bdk_wallet = { path = "../../crates/wallet" }
```

Comment:
Can we not depend on `bdk_wallet` for its changeset. Just redefine it here for now. I know it's friction and seems like a waste of time but the point of the `example_cli` examples is more to demonstrate to us as developers what you have to do to make a wallet with `bdk_chain` + the backends. There is of course still way too much you have to do but it's better not to hide it by depending on `bdk_wallet`.